src/main/scala/com.snowplowanalytics.snowplow.micro/SqliteStorage.scala

```diff
@@ -123,20 +123,25 @@ private[micro] class SqliteStorage(xa: Transactor[IO], maxEvents: Option[Int]) e
   }
 
   override def getColumnStats(columns: List[String]): IO[Map[String, ColumnStats]] = {
+    import SqliteStorage.COLUMN_STATS_SCAN_LIMIT
```

Comment:
This ensures we can fetch items for the column filter options _without_ blowing up the query execution time over large datasets.